Dockerfiles/Dockerfile.tests

```diff
@@ -1,18 +1,17 @@
 FROM golang:1.15-buster
 
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
+ 
+WORKDIR /go/src/github.com/CMSgov/bcda-app
```

Comment:
Is there caching advantages from moving this block up?

Reply:
Nope, just moving it up so I can access the tools.go file